src/SFA.DAS.ApprenticeFeedback.Application/Commands/CreateExitSurvey/CreateExitSurveyCommandResponse.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticeFeedback.Application.Commands.CreateExitSurvey
+{
+    public class CreateExitSurveyCommandResponse
+    {
+        public Guid? ApprenticeExitSurveyId { get; set; }
```

Comment:
Minor point, should this be nullable? if it succeeds, it's populated, if it fails, this response isn't instantiated?